addons/website/views/snippets/s_unveil.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_unveil" name="Unveil">
+    <section class="s_unveil pt64 pb64">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="13">
+                <div class="o_grid_item g-height-3 g-col-lg-12 col-lg-12" style="grid-area: 1 / 1 / 3 / 13;">
```

Comment:
The `g-height` class and the number of rows do not match, either set `g-height-2` or increase the grid area by one (I would do the second option, because the text barely fits in the grid area, considering its padding) :wink: